var/spack/repos/builtin/packages/py-petsc4py/package.py

```diff
@@ -20,6 +20,7 @@ class PyPetsc4py(PythonPackage):
     license("BSD-2-Clause")
 
     version("main", branch="main")
+    version("3.21.1", sha256="6ce1e1a45407da300c6869d0d9abe17b5b077424aa4895713642dda0bb19ab4e")
```

Comment:
This looks like a mistake - presumably it should be `3.21.2` and not a `3.21.1`.

Reply:
ops - thanks for catching it.

I have a fix at https://github.com/spack/spack/pull/44452